crates/orchestrator/src/tests/config.rs

```diff
@@ -178,12 +176,13 @@ impl TestConfigBuilder {
         } = self;
 
         let (starknet_client, server) = implement_client::init_starknet_client(starknet_client_type).await;
-        let alerts = implement_client::init_alerts(alerts_type, &settings_provider, &aws_config).await;
+        let alerts = implement_client::init_alerts(alerts_type, &settings_provider, provider_config.clone()).await;
```

Comment:
can we not pass a reference/?

Reply:
Done, https://github.com/madara-alliance/madara-orchestrator/pull/107/commits/2b0fb3c53125ea96a0bed03d61d726cf9c6f1714